au4/src/projectscene/iprojectscenecontroller.h

```diff
@@ -0,0 +1,22 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+#pragma once
+
+#include "modularity/imoduleinterface.h"
+#include "async/channel.h"
+#include "global/progress.h"
+#include "actions/actiontypes.h"
+
+namespace au::projectscene {
+class IProjectSceneController : MODULE_EXPORT_INTERFACE
```

Comment:
let's rename to IProjectSceneActionsController (and file name)

Reply:
fixed